include/rbdl/rbdl_config.h.cmake

```diff
@@ -74,4 +74,16 @@
 #  define RBDL_LOCAL RBDL_DLLLOCAL
 # endif // RBDL_BUILD_STATIC
 
+#ifdef __GNUC__
+#  define UNUSED(x) UNUSED_ ## x __attribute__((__unused__))
+#else
+#  define UNUSED(x) UNUSED_ ## x
+#endif
+
+#ifdef __GNUC__
+#  define UNUSED_FUNCTION(x) __attribute__((__unused__)) UNUSED_ ## x
+#else
+#  define UNUSED_FUNCTION(x) UNUSED_ ## x
+#endif
```

Comment:
Any reason to have this in a separate #ifdef section and not in the one above?

Reply:
Hi Martin,
thanks for your feedback. I moved the defines together below a single pragma.
Best
Manuel